content/en/docs/developer/e2e-guide.md

```diff
@@ -62,6 +72,27 @@ When testing locally, you may want to skip some setup steps to save time, especi
   ./test/e2e/run_test.sh --only-run-tests
   ```
 
-## How to add test case
+- Specifying a Pre-existing KinD Cluster:
+
+  ```bash
+  ./test/e2e/run_test.sh --cluster <KinD-Cluster-Name>
+  ```
+- Cleanup the KinD Cluster or Docker Registry After Tests
+
+  ```bash
+  ./test/e2e/run_test.sh --cleanup
+  ```
 
-TODO
+You can directly use go test command-line arguments when running the tests. For example, you can filter specific tests, or control other aspects of the testing process directly from the command line.
+
+- Selecting Specific Test Cases:
+
+  ```bash
+  ./test/e2e/run_test.sh --select-cases TestSpecificCase
+  ```
+
+- Controlling Test Verbosity
```

Comment:
This example is tedious, maybe could add an example that show how to repeat test cases multiple times.